### DISCUSSION TITLE
Install AdGuardHome on MacOS El Capitan

### DISCUSSION THREAD

**Discussion by Gihob** [AdguardTeam/AdGuardHome#6241] [Q&A]
Hi all,

I'm new on this forum and frankly, new to the issue at stake. I formatted the disk of a MacBook5.1, got it up and running again to OS X 10.11.6 (El Capitan) and want to use it as a dedicated AdGuard Home server.

I first tried the Automated install with
curl -s -S -L https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh | sh -s -- -v
but got certification-errors as shown in the screenshot
![Install MacBook 2023-09-16 at 3 44 39 pm](https://github.com/AdguardTeam/AdGuardHome/assets/87228458/c6cf18d7-24e2-4454-bd51-77d27cf66723)
Tried curl-options '-k' and '--insecure' in several ways, but without result.

Then tried the Manual way, downloaded the file AdGuardHome_darwin_amd64.zip and copied its content to the Applications folder. The next screenshot shows the result of the installation-attempt.
![Screen Shot 2023-09-20 at 12 12 26](https://github.com/AdguardTeam/AdGuardHome/assets/87228458/5f8361cb-cbad-48cd-96d7-b0865f8e6fe4)

Would highly appreciate if somebody on this forum could give me a hint or helping hand. 
Thanks a lot, Rob

**Comment by ainar-g** [comment]
Looks like your system might have outdated root CAs.  If you are able to download the ZIP, you can just follow the same procedure as described in the [manual update guide](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update-unix) starting with step 5.  Create the necessary directories as you go and call `./AdGuardHome -s install` before calling `./AdGuardHome -s start`.

**Comment by Gihob** [comment]
Don't know what 'root CAs' are, but the MacBook is from 2008.

Your solution IMHO is the same as what i described in the second half of my message under "tried the Manual way".
But then the system complains about "_clock_gettime". (see message)
Can this be solved/repaired?

**Reply by ainar-g** [reply]
Ah, right, apologies for not noticing.  And it seems like it can't be fixed without an OS update, unfortunately, since the stdlib of the Go programming language has been requiring macOS later than 10.11.6 since around early 2021, around the time v0.105 of AGH was released.

**Reply by Gihob** [reply]
Thank you Ainar for looking into my problem. I'm afraid that my old MacBook has outlived itself although still doing fine. To bad i can't use it as AdGuard HOME server, so i better stick to AdGuard DNS.
It's features are virtually the same, but no need to open browser-ports when away from home. (every disadvantage has it's advantage :-)
Thanks again and good luck. Rob
